### PR TITLE
Add new Example B7: Account takeover

### PR DESCRIPTION
**Pull request by damienbod**
fixes https://github.com/oauthstuff/draft-cross-device-security/issues/21

### CODE REVIEW

**Comment by PieterKas** [comment]
I think this is a valid attack. One open question is whether it is sufficiently different from the other examples. A second question is whether people should be using Verfiable Credentials for login use cases. @danielfett would love to get your opinion on this before merging.

**Comment by damienbod** [comment]
@danielfett  Any chance of a review?

**Comment by damienbod** [comment]
@PieterKas @danielfett Any move on this?

File: main.md [hunk 1]
```diff
@@ -252,6 +252,9 @@ An attacker initiates an employee onboarding flow and obtains a QR code from the
 
 ## Example B6: Illicit Session Transfer
 
+## Example B7: Account takeover
+The attacker creates a phishing website which will in real time create QR Codes from the original website and present these to the victim to invoke a wallet and present a verifiable credential attesting an existing identity. The attacker tries to get the victim to use the phishing website using an e-mail campaign etc. The victim scans the QR code on the phishing website, invokes their wallet and presents their credentials. Once the credentials are presented, the original session from the attackers device is updated with the victim's session.
```
Comment (danielfett) [inline main.md:256]:
```suggestion
An attacker wants to use some website which requires presentation of a verifiable credential for authentication. The attacker creates a phishing website which will in real time show log-in QR Codes from the original website and present these to the victim. The attacker tries to get the victim to use the phishing website using an e-mail campaign etc. The victim scans the QR code on the phishing website, invokes their wallet and presents their credentials. Once the credentials are presented, the original session from the attackers device is updated with the victim's session. 
```

**Review by danielfett — CHANGES_REQUESTED** [review]

File: main.md (hunk 1 above)
Comment (damienbod) [reply main.md]: Thanks for the review, updated

**Comment by damienbod** [comment]
@aaronpk @PieterKas Should be good now, no idea why the build fails, the new text has the same format as the other ones.

Greetings Damien

File: draft-ietf-oauth-cross-device-security.md [hunk 2]
```diff
@@ -258,6 +258,9 @@ An attacker initiates an employee onboarding flow and obtains a QR code from the
 ## Example B7: Illicit session transfer
 An attacker creates a message to all employees of a company, claiming to be from a trusted technology provider investigating a suspected security breach. They ask employees to send them the QR code typically used to transfer a session. The employee, eager to assist, initiates the process to transfer a session. They authenticate  and obtain a QR code and then send the QR code to the attacker. The attacker scans the QR code with their mobile phone and access the users data and resources.
 
+## Example B8: Account takeover
+An attacker wants to use some website which requires presentation of a verifiable credential for authentication. The attacker creates a phishing website which will in real time show log-in QR Codes from the original website and present these to the victim. The attacker tries to get the victim to use the phishing website using an e-mail campaign etc. The victim scans the QR code on the phishing website, invokes their wallet and presents their credentials. Once the credentials are presented, the original session from the attackers device is updated with the victim's session.
```
Comment (danielfett) [inline draft-ietf-oauth-cross-device-security.md:262]: Two more things - the first one is actually my mistake!

```suggestion
An attacker wants to use some website which requires presentation of a verifiable credential for authentication. The attacker creates a phishing website which will in real time capture log-in QR Codes from the original website and present these to the victim. The attacker tries to get the victim to use the phishing website using an e-mail campaign etc. The victim scans the QR code on the phishing website, invokes their wallet and presents their credentials. Once the credentials are presented, the original session from the attackers device is authenticated with the victim's credential.
```

**Review by danielfett — CHANGES_REQUESTED** [review]

File: draft-ietf-oauth-cross-device-security.md (hunk 2 above)
Comment (damienbod) [reply draft-ietf-oauth-cross-device-security.md]: Thanks for the feedback, update the text.

**Review by danielfett — APPROVED** [review]

**Comment by damienbod** [comment]
@danielfett Thanks!

**Comment by damienbod** [comment]
@PieterKas Can we merge this?

**Comment by PieterKas** [comment]
Merged

**Pull request merged**